spdmlib/src/responder/capability_rsp.rs

```diff
@@ -69,6 +69,20 @@ impl ResponderContext {
             SpdmGetCapabilitiesRequestPayload::spdm_read(&mut self.common, &mut reader);
         if let Some(get_capabilities) = get_capabilities {
             debug!("!!! get_capabilities : {:02x?}\n", get_capabilities);
+
+            #[cfg(feature = "mandatory-mut-auth")]
+            if !get_capabilities
+                .flags
+                .contains(SpdmRequestCapabilityFlags::MUT_AUTH_CAP)
```

Comment:
Do we have any check for requester?

Reply:
> Do we have any check for requester?

Do you mean check MUT_AUTH_CAP capability present for responder it self in responder side?
Or to check capabilities for responder in requester side?